web/models.py

```
@@ -816,6 +839,29 @@ class ForumReply(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
+    # Add the same methods to the ForumReply class
+    def upvote_count(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `upvote_count`

(ANN201)